conformance/tests/basic/inferencepool_accepted.yaml

```diff
@@ -0,0 +1,27 @@
+# Basic InferencePool for acceptance testing.
+# This manifest defines the minimal required fields to create a valid
+# InferencePool resource, which the InferencePoolAccepted test will use
+# to verify that the controller recognizes and accepts the resource.
+
+apiVersion: inference.networking.x-k8s.io/v1alpha2
+kind: InferencePool
+metadata:
+  # This name must match the 'poolNN' variable defined in the
+  # conformance/tests/basic/inferencepool_accepted.go test file.
+  name: inferencepool-basic-accepted
+  # This namespace should be one created by the base manifests.
+  namespace: default
+spec:
+  # --- Selector (Required) ---
+  # Selects the Pods belonging to this pool.
+  selector:
+    app: "vllm-llama3-8b-instruct"
+
+  # --- Target Port (Required) ---
+  # The port the model server container listens on.
+  targetPortNumber: 8000
+
+  # --- Extension Reference ---
+  # GKE-specific configuration reference.
+  extensionRef:
+    name: vllm-llama3-8b-instruct-epp
```

Comment:
Where is the EPP extension being deployed in the conformance suite? Additionally, the prefix for the EPP service should be consistent with the conformance backend. FOr example:

```sh
$ helm install $CONFORM_BACKEND ./config/charts/inferencepool \
  --set inferencePool.modelServers.matchLabels.app=$CONFORM_BACKEND_LABEL \
```

Reply:
Updated the test to use `infra-backend-v1-epp` to match the backend in the sample test.  

Regarding the EPP extension deployment, my current thought was that the extension will deployed manually by the person performing the test per design document 

```The setup and configuration of the base Kubernetes cluster (e.g., GKE) and the installation/configuration of the Gateway API Inference Extension implementation being tested (controller, GatewayClass, etc.) are prerequisites and outside the scope of the test execution logic.```